routes/controllers/docsController.js

```diff
@@ -1,6 +1,34 @@
+const source = require('rfr');
+const MarkdownIt = require('markdown-it');
+const fs = require('fs');
+
+const { setupCommands } = source('lib/scanner/commandHandler');
+const md = new MarkdownIt();
+
+const gettingStartedMD = fs.readFileSync('docs/GettingStarted.md', 'utf-8');
+const configurationsMD = fs.readFileSync('docs/Configurations.md', 'utf-8');
+const roleReactionsMD = fs.readFileSync('docs/RoleReactions.md', 'utf-8');
+const spotifyPluginMD = fs.readFileSync('docs/SpotifyPlugin.md', 'utf-8');
+const gettingHelpMD = fs.readFileSync('docs/GettingHelp.md', 'utf-8');
+
+function renderMarkdown(command) {
+    return (command.docs) ? md.render(command.docs) : null;
+}
+
 const DocsController = {
     get(req, res) {
-        res.render('docs');
+        const client = {
+        };
+        setupCommands(client);
+        const commands = client.commands.map((command) => renderMarkdown(command)).filter(Boolean);
```

Comment:
What is that filter doing?

Reply:
This filter is to account for commands missing the docs statement. renderMarkdown() will either return a string or null if docs is missing. filter(boolean) will keep all the strings and remove the null elements in the list

Reply:
okay thats cool, I haven't seen that `.filter(Boolean);` syntax. I should probably update a couple other spots where I have `.filter(i => i !== null);`